data/fields/healthcare/speciality.json

```diff
@@ -1,5 +1,94 @@
 {
     "key": "healthcare:speciality",
     "type": "semiCombo",
-    "label": "Specialties"
+    "label": "Specialties",
+    "strings": {
+        "options": {
+            "abortion": "Abortion",
+            "acupuncture": "Acupuncture",
+            "allergology": "Allergy & Asthma",
+            "anaesthetics": "Anesthesiology",
+            "angiology": "Angiology",
+            "anthroposophical": "Anthroposophic Medicine",
+            "applied_kinesiology": "Applied Kinesiology",
+            "aromatherapy": "Aromatherapy",
+            "ayurveda": "Ayurveda",
+            "behavior": "Behavior Therapy",
+            "biochemistry": "Biochemistry",
+            "biology": "Biology",
+            "blood_check": "Blood Test",
+            "body": "Body Therapy",
+            "cardiology": "Cardiology",
+            "cardiothoracic_surgery": "Cardiothoracic Surgery",
+            "child_psychiatry": "Child and Adolescent Psychiatry",
+            "chiropractic": "Chiropractic",
+            "clinical_pathology": "Clinical Pathology",
+            "community": "Community and Public health",
+            "depth": "Depth Psychology",
+            "dermatology": "Dermatology",
+            "dermatovenereology": "Dermatovenereology",
+            "diabetology": "Diabetology",
+            "diagnostic_radiology": "Diagnostic Radiology",
+            "emergency": "Emergency Medicine",
+            "endocrinology": "Endocrinology",
+            "fertility": "Fertility",
+            "gastroenterology": "Gastroenterology",
+            "general": "General Practitioner",
+            "geriatrics": "Geriatric Medicine",
+            "gynaecology": "Obstetrics and Gynecology",
+            "haematology": "Hematology",
+            "hepatology": "Hepatology",
+            "herbalism": "Herbalism",
+            "homeopathy": "Homeopathy",
+            "humanistic": "Humanistic Psychology",
+            "hydrotherapy": "Hydrotherapy",
+            "hypnosis": "Hypnosis",
+            "infectious_diseases": "Infectious Disease",
+            "intensive": "Intensive Care",
+            "internal": "Internal Medicine",
+            "dental_oral_maxillo_facial_surgery": "Dental, Oral & Maxillofacial Surgery",
+            "naturopathy": "Naturopathy",
+            "neonatology": "Neonatology",
+            "nephrology": "Nephrology",
+            "neurology": "Neurology",
+            "neuropsychiatry": "Neuropsychiatry",
+            "neurosurgery": "Neurosurgery",
+            "nuclear": "Nuclear Medicine",
+            "occupational": "Occupational Medicine",
+            "oncology": "Oncology",
+            "ophthalmology": "Ophthalmology",
+            "orthodontics": "Orthodontics",
+            "orthopaedics": "Orthopedic Surgery",
+            "osteopathy": "Osteopathy",
+            "otolaryngology": "Ear, Nose and Throat",
```

Comment:
Thank you for using the colloquial name – I don’t think I’ve ever seen a medical facility signposted as specializing in “otolaryngology” or “otorhinolaryngology”, so I would have a difficult time finding this tag otherwise.